extra/rp2040/cyw43/cyw43_spi.fs

```diff
@@ -223,9 +225,8 @@ begin-module cyw43-spi
       sm bit pio sm-disable
 
       \ Set up the PIO program
-      pio cyw43-pio-program p-size alloc-piomem { base-addr }
-      sm pio cyw43-pio-program base-addr setup-prog
-      base-addr sm pio sm-addr!
+      sm pio cyw43-pio-program pio-addr setup-prog
```

Comment:
I thought we were getting rid of `pio-addr`, and just leaving it in some constructors to avoid breaking other people's code which uses the CYW43439 driver, and indeed it does not appear to be defined anywhere in this word. Rather, the original code appears to be correct, and should be left in place.

Reply:
Okay, gotcha. I missed the definition of PIO-ADDR because it was not shown in the diff (and because I forgot to expand all of the non-difference parts). I will merge in the code now.